nest/06.glossary.md

```diff
@@ -0,0 +1,182 @@
+The cache has to be kept up-to-date in order to install the latest versions of the packages.
+
+* Category
+
+[//]: # (TODO: add links to the 'repository' and 'package' entry of this glossary)
+Inside each [repository](), there are several categories.
+The [packages]() are split between the repositories and their categories.
+Each package name must be unique in a category of a repository, but there can be several packages with the same name spread in several categories.
+
+## D
+[//]: # (TODO: add link to the section 1.2)
+* Dependency [section 1.2]()
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+A dependency is a [package]() A that is required by a package B, so that the package B can function properly.
+
+[//]: # (TODO: add link to the section 4.1.3)
+* Dependency graph/Dependency tree [section 4.1.3]()
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+The dependency graph is a directed graph (i.e. a graph with directed connections) representing the dependencies between [packages]().
+## E
+
+## F
+[//]: # (TODO: add link to the chapter 4)
+* `finest` [chapter 4]()
+
+`finest` is the binary of Nest targeted towards experienced users.
+
+* `finest group create <name> [--parent <group>]`
+
+[//]: # (TODO: add link to the 'group' entry of this glossary)
+The command of `finest` to create a [group](), with `@root` as a parent by default.
+
+* `finest group list`
+
+[//]: # (TODO: add link to the 'group' entry of this glossary)
+The command of `finest` to list all [groups]() on the OS.
+
+* `finest group remove <name> [--parent <group>]`
+
+[//]: # (TODO: add link to the 'group' entry of this glossary)
+The command of `finest` to remove a [group](), with `@root` as a parent by default.
+
+* `finest merge`
+
+[//]: # (TODO: add link to the 'Dependency graph' and 'transaction' entries of this glossary)
+The command of `finest` to merge the [dependency graph]() `finest` altered with the [transactions]() you applied, with the dependency graph currently in use by the OS.
+
+* `finest requirement add <requirement> [--parent <group>]`
+
+[//]: # (TODO: add link to the 'requirement' and 'group' entries of this glossary)
+The comand of `finest` to add a [requirement]() to a [group](), by default `@root`.
+
+* `finest requirement remove <requirement> [--parent <group>]`
+
+[//]: # (TODO: add link to the 'requirement' and 'group' entries of this glossary)
+The command of `finest` to remove a [requirement]() from a [group](), by default `@root`.
+
+## G
+[//]: # (TODO: add link to the section 4.1.4)
+* Groups [section 4.1.4]()
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+Groups are a way to group [packages]() together, in order to install and uninstall them in a more efficient manner.
+## H
+
+## I
+
+## J
+
+## K
+
+## L
+
+## M
+* Mirror
+
+A mirror is a server with a copy of all the data from a main server.
```

Comment:
```suggestion
A mirror is an alternative server with a copy of all the data from a main server. They are used when the main server is out of service.
```

Reply:
If you are going that direction, maybe also mention that they can speed up download speed by being closer geographically.